app/db/models/app/users.py

```diff
@@ -0,0 +1,36 @@
+import sqlalchemy as sa
+from sqlalchemy import PrimaryKeyConstraint
+
+from app.db.session import Base
+
+
+class Admin(Base):
+    """Table of admin users in the system."""
+
+    __tablename__ = "admin"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    name = sa.Column(sa.String)
+    email = sa.Column(sa.String, unique=True, index=True, nullable=False)
+    password = sa.Column(sa.String)
+
+
+class Organisation(Base):
+    __tablename__ = "organisation"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    name = sa.Column(sa.String)
+    description = sa.Column(sa.String)
+    organisation_type = sa.Column(sa.String)
+
+
+class AdminOrganisation(Base):
+    """Link table for admin and organisation.  """
+
+    __tablename__ = "admin_organisation"
+
+    admin_id = sa.Column(sa.Integer, sa.ForeignKey(Admin.id), nullable=False)
+    organisation_id = sa.Column(sa.Integer, sa.ForeignKey(Organisation.id), nullable=False)
+    job_title = sa.Column(sa.String)
+
+    PrimaryKeyConstraint(admin_id, organisation_id)
```

Comment:
A few questions here:
 - Do we want to give this PK a name, or are we happy yo just use the default?
 - Is this how we want to specify PK constraints for join tables, because it doesn't seem to be consistent with e.g. `PhysicalDocumentLanguage` (I'm not saying it doesn't make sense, it really does, but consistency is good too :)
 - Does defining the PK this way introduce any difficulties?

Reply:
FTR - talked about this and we're going to make it consistent - removing the id and using the two columns in a link table